common/lib/api-client/index.js

```diff
@@ -55,6 +61,18 @@ module.exports = function () {
   insertRequestMiddleware(requestTimeout(API.TIMEOUT))
   insertRequestMiddleware(requestHeaders)
   insertRequestMiddleware(requestInclude)
+  insertRequestMiddleware({
+    name: 'clean-original-request',
+    req: payload => {
+      const { jsonApi } = payload
+
+      if (jsonApi._originalReq) {
+        delete jsonApi._originalReq
+      }
+
+      return payload
+    },
+  })
```

Comment:
I am definitely missing something here, but what is this achieving?

Reply:
If it's the block of code I just removed, the answer is nothing :). (I based this off of an initial investigation by Dom and had forgotten to remove it)